js-webapp-async/newtonraphson.hpp

```diff
@@ -1,15 +1,11 @@
 #ifndef H_NEWTONRAPHSON_H
 #define H_NEWTONRAPHSON_H
 
-#include <string>
-
-namespace rootfinding {
-   class NewtonRaphson {
-      public:
-         NewtonRaphson(double tolerancein);
-         double solve(double xin);
-      private:
-         double tolerance;
-  };
-}
+class NewtonRaphson {
+   public:
+      NewtonRaphson(double tolerancein);
```

Comment:
The variables should be float instead of double. Also the var name changed.

Reply:
Replace tolerancein with tolerance_in and xin with initial_guess

Reply:
I fixed these in PR #75